syntax = "proto3";

package chatroom.msgpb;

message GetJoinedGroups {
    int32 user_id = 1;
}